### DISCUSSION TITLE
Page résultat : Normalisation des impacts environnementaux

### DISCUSSION THREAD

**Discussion by margaux-escande** [cnumr/EcoIndex#147] [Ideas]
Changer ce texte pour qu'il soit dynamique et donc directement donner le nombre de KM et de Douche 

Si votre page émet 1,7g CO2e et consomme 2,5cl d’eau, cela signifie que pour 1000 visiteurs mensuels, l’empreinte est de 1,7kg de CO2e et 25 litres d’eau par mois – soit un trajet de 5,5km en voiture et une douche de 4 minutes.

**Comment by AmauryVanEspen** [comment]
Inspirant globalement cette initiative d'index,
quid de la question du coût complet en ACV ?
